### PR TITLE
Fetch photos from CMS

### PR DESCRIPTION
**Pull request by rodonguyen**
image
image

### CODE REVIEW

File: src/types/app.ts [hunk 1]
```diff
@@ -54,4 +54,25 @@ export type Sponsor = {
   ];
 };
 
+export type Photo = {
+  data: [
```
Comment (bitcoinbrisbane) [inline src/types/app.ts:58]: We dont need data.  Photo.data doesnt make sense.

**Review by bitcoinbrisbane — CHANGES_REQUESTED** [review]
One type issue

File: src/types/app.ts (hunk 1 above)
Comment (rodonguyen) [reply src/types/app.ts:58]: That's what we [received from CMS](https://cms.dltx.io/api/photos?populate=*). You did the same with Sponsors. 
No fix, then?

Comment (rodonguyen) [reply src/types/app.ts]: Made changes

File: src/components/NextMeetup.tsx [hunk 2]
```diff
@@ -116,7 +116,7 @@ export const NextMeetup = () => {
             meetup.com/bitcoinbrisbane
           </a>
         </p>
-      </div> */}
+      </div> */}
```
Comment (bitcoinbrisbane) [inline src/components/NextMeetup.tsx:119]: Need to lint / prettier

**Review by bitcoinbrisbane — CHANGES_REQUESTED** [review]
lint?

File: src/components/NextMeetup.tsx (hunk 2 above)
Comment (rodonguyen) [reply src/components/NextMeetup.tsx]: Done

**Review by bitcoinbrisbane — APPROVED** [review]

**Pull request merged**